vhdbuilder/packer/test/run-test.sh

```diff
@@ -6,7 +6,7 @@ WIN_SCRIPT_PATH="windows-vhd-content-test.ps1"
 TEST_RESOURCE_PREFIX="vhd-test"
 TEST_VM_ADMIN_USERNAME="azureuser"
 
-set +x
```

Comment:
what is the purpose to remove this line?

Reply:
Added back.